nat/nat.go

```diff
@@ -34,6 +34,9 @@ func NewPort(proto, port string) Port {
 }
 
 func ParsePort(rawPort string) (int, error) {
+	if len(rawPort) == 0 {
+		return 0, nil
```

Comment:
^^ Just to be sure, this was alright? (In case someone visits this at a later stage)